initial_bot/mentor_bot_test.py

```diff
@@ -1,32 +1,65 @@
 # bot.py
+import pdb
 import sys
 import os
 import random
 import discord
 import argparse
 from discord.ext import commands
+from discord.utils import get
+
 
 def parse_arguments():
     parser = argparse.ArgumentParser()
-    parser.add_argument('-t', '--token', dest = 'token', help = "If you don't have the token please ask Eran to send it to you")
-    parser.add_argument('-g', '--guild_name', dest = 'guild', default = "mentorbot_test")
+    parser.add_argument('-t', '--token', dest='token',
+                        help="If you don't have the token please ask Eran to send it to you")
+    parser.add_argument('-g', '--guild_name', dest='guild',
+                        default="mentorbot_test")
     return parser.parse_args()
 
+
 def main():
-    parsed_args = parse_arguments()
-    if parsed_args.token == None:
-        print("Please try again, but this time enter a token as an argument")
-        sys.exit()
+    # parsed_args = parse_arguments()
+    # if parsed_args.token == None:
+    #     print("Please try again, but this time enter a token as an argument")
+    #     sys.exit()
 
     intents = discord.Intents.all()
     discord.member = True
-    bot = commands.Bot(command_prefix='!', intents = intents)
+    bot = commands.Bot(command_prefix='!', intents=intents)
 
     @bot.event
     async def on_ready():
         print(f'{bot.user.name} has connected to Discord!')
 
-    # Send a gretting to members upon joining the server
+    @bot.command(name='private')
+    async def make_channel(ctx, member: discord.Member, member1: discord.Member, member2: discord.Member, room_name):
```

Comment:
We cant control the info we are getting, this comes in objects in this case because of how Discord built it.
I am unsure if we should change our approach at the moment or not, @erantr1 ?